cpp/arcticdb/CMakeLists.txt

```diff
@@ -524,12 +528,13 @@ endif()
 # on their origin (i.e. vendored source or packages on conda-forge).
 if(${ARCTICDB_USING_CONDA})
 
-    # We must use dynamic libraries.
+    # We must use dynamic libraries, except for xxHash, which must be statically linked.
+    # In non-conda builds, we inline the functions so it does not need linking at all.
     list (APPEND arcticdb_core_libraries
-        xxHash::xxHash
         pcre
         msgpack-c
         ${LMDB_LIBRARIES}
+        xxHash
     )
```

Comment:
Was this change needed because of vcpkg's distribution?

Reply:
No, you can use either, I'd just removed it and then put it back with the alternative name. Reverted now.